certified-connectors/Iconfirm/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4257 @@
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/CustomTaskData"
+              }
+            }
+          }
+        },
+        "operationId": "FindCustomTasks",
+        "description": "Retrieves a list of all custom tasks registered."
+      }
+    },
+    "/v1/customtasks/byId/{id}": {
+      "get": {
+        "tags": [
+          "CustomTasksApi"
+        ],
+        "summary": "Get Custom Task by Id",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "description": "Id of the customtask you want to retrieve.",
+            "required": true,
+            "type": "integer",
+            "format": "int32",
+            "x-ms-summary": "Id of the customtask you want to retrieve.",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Task retrieved successfully",
+            "schema": {
+              "$ref": "#/definitions/CustomTaskData"
+            }
+          }
+        },
+        "operationId": "GetCustomTaskById",
+        "description": "Gets a single custom task with the given Id."
+      }
+    },
+    "/v1/customtasks/byUuid/{uuid}": {
+      "get": {
+        "tags": [
+          "CustomTasksApi"
+        ],
+        "summary": "Get Custom Task by Uuid",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "uuid",
+            "description": "Uuid of the customtask you want to retrieve.",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Uuid of the customtask you want to retrieve.",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Task retrieved successfully",
+            "schema": {
+              "$ref": "#/definitions/CustomTaskData"
+            }
+          }
+        },
+        "operationId": "GetCustomTaskByUuid",
+        "description": "Get a single custom task by the given Uuid."
+      }
+    },
+    "/v1/customtasks/updateStatus": {
+      "post": {
+        "tags": [
+          "CustomTasksApi"
+        ],
+        "summary": "Update Status for Custom Task",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "taskUuid",
+            "description": "Uuid of the customtask you want to retrieve.",
+            "type": "string",
+            "x-ms-summary": "Uuid of the customtask you want to retrieve."
+          },
+          {
+            "in": "query",
+            "name": "newStatus",
+            "description": "The new status to be set for the task.",
+            "type": "string",
+            "format": "",
```

Comment:
@iconfirm-development Empty property values can be removed until unless you have a specific reason to keep them.

Applies the same comment anywhere in the connector artifacts

Reply:
@sriyen-msft it's now removed, searched for empty strings in definition and that seems to be the last one